server/controllers/user.js

```diff
@@ -0,0 +1,8 @@
+const welcome = (req, res) => {
+      return res.status(200).json({
+          status: 200,
+          message: 'Welcome to phantom, a platform to facilitate the transportation mode in town!.'
+      });
+  };
+
+  export default welcome;
```

Comment:
instal eslint on your vscode so that new lines can be added to every end of your file